.github/workflows/code_formatting_checks.yml

```diff
@@ -0,0 +1,49 @@
+name: Code formatting
+
+# Scheduled at 11:30 PM UTC (05:00 AM IST)
+on: push
+
+jobs:
+  fix_code_formatting:
+    runs-on: [ubuntu-latest]
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Set up JDK 1.8
+        uses: actions/setup-java@v1
+        with:
+          java-version: 1.8
+
+      - name: Fix code formatting issues
+        run: ./gradlew spotlessApply
+
+      - name: Commit and Push
+        run: .github/scripts/commit_and_push.sh "${{secrets.USER_NAME}}" "${{secrets.USER_EMAIL}}"
+
+      - name: Create PR
+        uses: actions/github-script@0.3.0
+        with:
+          github-token: ${{secrets.GITHUB_TOKEN}}
+          script: |
+            var monthNames = ["Jan", "Feb", "Mar", "Apr", "May", "Jun", "Jul", "Aug", "Sep", "Oct", "Nov", "Dec"];
+            var date = new Date();
+            var monthName = monthNames[date.getMonth()];
+            var day = `${d.getDate()}`.padStart(2, 0);
+            day = `${monthName}-${day}-${date.getFullYear()}`
+            var branchName = `bot/${day}/code-formatting`
+
+            try {
+                const branch = await github.repos.getBranch({
+                    ...context.repo,
+                    branch: branchName
+                });
+                await github.pulls.create({
+                    ...context.repo,
+                    title: `Code formatting fixes on ${day}`,
```

Comment:
Can we prefix the PR title with `[BOT]` to indicate that this is an automated check? Maybe we can also skip adding the 'on' in the title.

This would make the title look something like `[BOT] Code formatting fixes Jan-05-2019`. What do you think?

Reply:
Yeah, this makes more sense. Changed it